syntax = "proto3";

enum MessageType {
  Ping = 0;
  
  GenerateThumbnails = 1;

}

message GenerateThumbnailsRequest {
    string path = 1;
}
